### PR TITLE
cmd: improve app-shell error output 

### LINKED ISSUE #1115: cmd: improve app-shell error output
Currently `app-shell` is returning the http error:

    $ tsuru app-shell -a tsuru-dashboard3
    Error: HTTP/1.1 404

The command should return an user friendly error message.

### PR DESCRIPTION
**Pull request by wpjunior**
Fix #1115.

### CODE REVIEW

File: cmd/shell.go [hunk 1]
```diff
@@ -59,6 +58,7 @@ func (c *ShellToContainerCmd) Run(context *Context, client *Client) error {
 			signal.Notify(sigChan, syscall.SIGINT, syscall.SIGTERM, syscall.SIGKILL, syscall.SIGQUIT)
 		}
 	}
+
```
Comment (cezarsa) [inline cmd/shell.go]: Please, remove this empty line

File: cmd/shell.go [hunk 2]
```diff
@@ -109,7 +108,14 @@ func (c *ShellToContainerCmd) Run(context *Context, client *Client) error {
 	}
 	matches := httpHeaderRegexp.FindAllStringSubmatch(readStr, -1)
 	if len(matches) > 0 && len(matches[0]) > 1 {
-		return errors.New(strings.TrimSpace(readStr))
+		httpError, _ := strconv.Atoi(matches[0][1])
+		var message string
+		if (httpError == http.StatusNotFound) {
+			message = "App "+appName+" not found"
```
Comment (fsouza) [inline cmd/shell.go]: Please use fmt.Sprintf instead of this concatenation.

Thanks for contributing!

**Comment by wpjunior** [comment]
OK:
https://github.com/tsuru/tsuru/blob/master/cmd/shell.go#L83

**Comment by fsouza** [comment]
This is different because we're concatenating only in the end of the string.

**Comment by wpjunior** [comment]
Done!

**Comment by fsouza** [comment]
Thanks!

**Pull request merged**